srv/salt/_modules/cephimages.py

```diff
@@ -6,20 +6,30 @@
 from glob import glob
 from subprocess import Popen, PIPE
 
+def _is_installed(name):
+    try:
+        devnull = open(os.devnull)
+        Popen(['type', name], stdout=devnull, stderr=devnull).communicate()
```

Comment:
Can't you use the `__salt__['pkg.info_installed']` module function?

The code would look like this:
```
try:
    res = __salt__['pkg.info_installed`](name)
    return name in res
except Exception:
    return False
```
Then for the case of checking the availability of the `rados` command, you can call this with `_is_installed("ceph-common")`